### PR TITLE
Adding support for describing a stack

### PR DESCRIPTION
**Pull request by jbank**
I also upgraded to the latest aws java version 1.9.24

### CODE REVIEW

File: src/main/java/com/amazonaws/ant/cloudformation/DescribeStackTask.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
+package com.amazonaws.ant.cloudformation;
```
Comment (fulghum) [inline src/main/java/com/amazonaws/ant/cloudformation/DescribeStackTask.java]: Go ahead and throw in the standard Apache 2.0 header comment that we use in other Java source files in this project.

**Comment by fulghum** [comment]
Hey Jesse, thanks for the contribution.  Can you confirm that you're releasing this code under the Apache  2.0 license?

**Comment by jbank** [comment]
Sounds good. Can we please get a release with this?

**Comment by david-at-aws** [comment]
Working on a release now.

**Comment by david-at-aws** [comment]
[boomshakalaka](https://github.com/awslabs/aws-ant-tasks/releases/tag/v1.2.3)

**Pull request merged**